client/components/open/forms/components/FirstSubmissionModal.vue

```diff
@@ -105,4 +104,10 @@ const helpLinks = computed(() => {
   ]
 })
 
-</script>
+const trackOpenDbClick = () => {
+  const submissionsUrl = `/forms/${props.form.slug}/submissions`
```

Comment:
Have you tested this...? Not working not the right URL. 

To fix:
- Create a new prop on Model form submission_url (like we did here https://github.com/JhumanJ/OpnForm/blob/8431ad491f6f1112c2de3037073d8ea748eef913/api/app/Models/Forms/Form.php#L147-L154)
- If user is logged in and form owner add this prop to FormResource (like we did here https://github.com/JhumanJ/OpnForm/blob/8431ad491f6f1112c2de3037073d8ea748eef913/api/app/Http/Resources/FormResource.php#L28-L29)
- use this to redirect